input/input_overlay.c

```diff
@@ -338,7 +336,7 @@ static void input_overlay_poll(
       }
    }
 
-   if (!out->buttons)
+   if (bits_any_set(out->buttons.data, ARRAY_SIZE(out->buttons.data)))
```

Comment:
This is backwards to the original code - fixed this in second patch which unhelpfully didn't combine with a force push.